components/library/src/content/page.rs

```diff
@@ -82,13 +82,12 @@ pub struct Page {
     pub lang: String,
     /// Contains all the translated version of that page
     pub translations: Vec<DefaultKey>,
-    /// Contains the internal links that have an anchor: we can only check the anchor
-    /// after all pages have been built and their ToC compiled. The page itself should exist otherwise
-    /// it would have errored before getting there
-    /// (path to markdown, anchor value)
-    pub internal_links_with_anchors: Vec<(String, String)>,
-    /// Contains the external links that need to be checked
-    pub external_links: Vec<String>,
+    /// The list of all internal links (as path to markdown file), with optional anchor fragments.
+    /// We can only check the anchor after all pages have been built and their ToC compiled.
+    /// The page itself should exist otherwise it would have errored before getting there.
+    pub internal_links: Vec<(String, Option<String>)>,
+    /// The list of all links to external webpages. They can be validated by the `link_checker`.
+    pub external_web_links: Vec<String>,
```

Comment:
why did you change that name?

Reply:
While going through the existing logic, I realized there is a filtering step applied when building this collection, so that it effectively only holds HTTP(S) links.
I was mildly surprised that is field is only a subset of "all the external links in the page", thus I aligned the name and docstrings to mention web/webpages (as noted in the commit message).

This rename is split in its own commit, so it can be quickly dropped if you prefer so.

Reply:
Let's drop that change then, I don't think anyone expects phone numbers etc to be part of that list.